src/scss/skin-modern/components/_subtitleoverlay-cea608.scss

```diff
@@ -0,0 +1,7 @@
+.#{$prefix}-ui-subtitle-overlay {
+  .#{$prefix}-cea608 {
```

Comment:
Change to `&.#{$prefix}-cea608 ..#{$prefix}-ui-subtitle-label` so you can apply the `cea608` directly to the `SubtitleOverlay` (see comment below)